src/neptune/management/internal/api.py

```diff
@@ -592,18 +592,15 @@ def invite_to_workspace(
             Note: To keep your token secure, use the NEPTUNE_API_TOKEN environment variable rather than placing your
             API token in plain text in your source code.
         role: The workspace role that is to be granted to the invited user.
-        You can choose between the following values:
-        - Administrator: `WorkspaceMemberRole.ADMIN`
-        - Member: `WorkspaceMemberRole.MEMBER`
+            You can choose between the following values: "admin" and "member".
```

Comment:
Don't we also want to encourage using the enum?

Reply:
Yes, my plan is to do it in the docs. Here I think it's better to showcase the most straightforward option.